src/common/schedulers/slurm_commands.py

```diff
@@ -43,8 +43,8 @@
     "cpus-per-tres",
 ]
 SQUEUE_FIELD_STRING = ",".join([field + ":{size}" for field in _SQUEUE_FIELDS]).format(size=SQUEUE_FIELD_SIZE)
-SCONTROL = "/opt/slurm/bin/scontrol"
-SINFO = "/opt/slurm/bin/sinfo"
+SCONTROL = "sudo /opt/slurm/bin/scontrol"
+SINFO = "sudo /opt/slurm/bin/sinfo"
```

Comment:
Minor point: for sure we need sudo for scontrol. I don't believe we need it for sinfo. Here are the usages I could find:

```python
show_all_nodes_command = f"{SINFO} -h -p {partition_name} -N -o %N"
show_power_down_nodes_command = f"{SINFO} -h -p {partition_name} -t power_down,powering_down -N -o %N"
show_down_nodes_command = f"{SINFO} -h -p {partition_name} -t down -N -o %N"
show_all_nodes_command = f"{SINFO} -h -p {partition_name} -o %N"
```

I'd add a comment stating that sudo isn't strictly needed for sinfo, but I'd leave it on there to account for any changes we make in the future that do make it necessary.

Reply:
Thanks you for the heads up. 
I think it would be better to remove sudo execution from every command that does not strictly require it, for sake fo security.
Furthermore, if we add a comment today that we execute it with sudo there is the risk for such a comment to become obsolete without being removed.